recipe/meta.yaml

```diff
@@ -155,6 +157,8 @@ outputs:
         # having not been migrated yet causing conflicts with the "libgdal" output
         # See: https://github.com/conda-forge/gdal-feedstock/pull/415
         - hdf5
+        # temporary workaround while ffi is migrating
+        - libffi
```

Comment:
@ocefpaf and @conda-forge/gdal, does this (and the corresponding dependency above) seem like an acceptable solution or do you foresee dangerous side effects?

Reply:
I think it'll benefit everybody if someone could send a PR to python-feedstock's `3.7, 3.8, 3.9` to update libffi

Reply:
@xylar and @isuruf I'll try to sent the PRs to older Pythons (finger crossed). Meanwhile let's merge this!

Thanks so much for your help! I was completely lost debugging this from 10 min per day only :-/